### PR TITLE
Fix DLC notebook to account for new change to element_DLC

### PR DESCRIPTION
**Pull request by sidhulyalkar**
- Change keys associated with inserting new PoseEstimationTask according to new changes in element_dlc
- Add population of model.BodyPart and model.Model.BodyPart tables into insert_new_dlc_model function

### CODE REVIEW

File: workflow/pipeline/dlc.py [hunk 1]
```diff
@@ -94,26 +94,24 @@ def insert_new_dlc_model(project_path, paramset_idx=None, model_prefix="", model
...
+                model_description=model_description,
+                model_prefix=model_prefix,
+                paramset_idx=paramset_idx,
+                prompt=prompt,
+                params=None,
+            )
+            
+            if model.BodyPart.extract_new_body_parts(dlc_config, verbose=False).size > 0:
+                model.BodyPart.insert_from_config(dlc_config, prompt=prompt)
+            model.Model.BodyPart.insert((model_name, bp) for bp in dlc_config["bodyparts"])
```
Comment (ttngu207) [inline workflow/pipeline/dlc.py]: This block is already included in the `model.Model.insert_new_model()`

File: workflow/pipeline/dlc.py [hunk 2]
```diff
@@ -94,26 +94,24 @@ def insert_new_dlc_model(project_path, paramset_idx=None, model_prefix="", model
...
-            "project_path": project_path.relative_to(root_data_dir).as_posix(),
-            "paramset_idx": paramset_idx,
-            "config_template": dlc_config,
-        }
-        model.Model.insert1(model_dict)
-        
-        if model.BodyPart.extract_new_body_parts(dlc_config, verbose=False).size > 0:
-            model.BodyPart.insert_from_config(dlc_config, prompt=prompt)
-        model.Model.BodyPart.insert((model_name, bp) for bp in dlc_config["bodyparts"])
+        if dlc_config["snapshotindex"] == -1:
```
Comment (ttngu207) [inline workflow/pipeline/dlc.py:97]: Does this mean only models with `snapshotindex` of `-1` get inserted?

Comment (sidhulyalkar) [reply workflow/pipeline/dlc.py:97]: Ah yes, fixed this in the last commit

File: workflow/pipeline/dlc.py (hunk 1 above)
Comment (sidhulyalkar) [reply workflow/pipeline/dlc.py]: Fixed in last commit

**Pull request merged**